### PR TITLE
Markup.ml 0.8.2 - standards-compliant HTML parser

### PR DESCRIPTION
**Pull request by aantron**
- Parse `<noscript>` and `<script>` fragments in context of `<body>` by default, rather than `<head>` (aantron/markup.ml#48, prompted Daniil Baturin).
- Change license to MIT (aantron/markup.ml@89bee81).

cc @dmbaturin

### CODE REVIEW

File: packages/markup/markup.0.8.2/opam [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "uutf" {>= "1.0.0"}
+
+  "bisect_ppx" {dev & >= "2.0.0"}
+  "ounit" {dev}
+]
+# Markup.ml implicitly requires OCaml 4.02.3, as this is a contraint of Dune.
+
+pin-depends: [
+  ["bisect_ppx.git" "git+https://github.com/aantron/bisect_ppx.git"]
+]
```
Comment (kit-ty-kate) [inline packages/markup/markup.0.8.2/opam]: I don't think a released package should have a pin-depends field. Users might pin this package if they just want this version, it doesn't have to be the dev version.

Comment (dmbaturin) [reply packages/markup/markup.0.8.2/opam]: I also found it odd when installing a dev version for testing. Why is it needed?

Comment (aantron) [reply packages/markup/markup.0.8.2/opam]: I've removed the field.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**